src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="star star--0">
```

Comment:
please, name it starts--0 as mentioned in a task and apply the same naming pattern for the rest of the blocks. Some of your testing are failed and it might be the reason.